Lib/idlelib/idle_test/test_squeezer.py

```diff
@@ -82,6 +92,15 @@ def test_several_lines_different_lengths(self):
 
 class SqueezerTest(unittest.TestCase):
     """Tests for the Squeezer class."""
+
+    @classmethod
+    def setUpClass(cls):
+        cls.font_family = get_font_family()
+
+    @classmethod
+    def tearDownClass(cls):
```

Comment:
This seems unnecessary.

Reply:
Probably true -- at the moment.  But this will be needed if root is created in setUpClass.